### ISSUE TITLE
destruction

### ISSUE THREAD

**Issue by arademaker** [globalwordnet/english-wordnet#186]
00217014-n devastation, destruction | (the termination of something by causing so much damage to it that it cannot be repaired or no longer exists) - noun.act

14562960-n destruction, end, death | (a final state; "he came to a bad end"; "the so-called glorious experiment came to an inglorious end") - noun.state

07334490-n demolition, destruction, wipeout | (an event (or the result of an event) that completely destroys something) - noun.event

I claim that in the last sense, in the lexicographer file noun.event , the gloss should not contain the `(or the result of an event)`. The resulting reading would clash with the previous sense in the noun.state lexicographer file.

**Comment by restinplace** [comment]
TL;DR -- an argument can be made for the status quo, but try to avoid building more quo with this status. 

This is one of WordNet's occasional hair-splitting decisions, which I always assume were prompted by wanting to account for some common collocation (e.g. "scene of destruction#n#3", which did not have a corpus entry).   I think the general phenomena are due to alternations between:
- a specific vs. abstract instance, and
- an action, and the result of the action.

These sometimes contrast conceptually in isolation, but not semantically in a given context (like the "samba" of "samba song" -- does it derive from verb or noun?), and are not always given sense distinctions.  For example score#v#1 does not distinguish between the individual scoring act ("He shoots ... he scores!') and a team's acquisition of points ("the team scored 3 goals in the second half").  But they are different acts -- when I score so does the team, but not vice versa.

Showing the WN corpus frequency:
- (8)<noun.act>S: (n) destruction#1, devastation#5 (the termination of something by causing so much damage to it that it cannot be repaired or no longer exists)
- (6)<noun.event>S: (n) destruction#2, demolition#1, wipeout#1 (an event (or the result of an event) that completely destroys something)
- <noun.state>S: (n) end#6, destruction#3, death#7 (a final state) "he came to a bad end"; "the so-called glorious experiment came to an inglorious end"

For example:
- the destruction#n#1 of Carthage by nuclear explosion == a (concrete) act
- the eve of destruction#n#2, the threat of environmental destruction#n#2 == an (abstract) event, or the result of an event
- a scene of destruction#n#3 == a final state (after destruction#n#1 or destruction#n#2)

- Threats of environmental destruction#n#2, such as destruction#n1 of the rain forest by burning, contribute to humanity's imminent state of destruction#n#3.

That said, please let me stress that these are awfully fine distinctions that do not always enjoy sole dominion over a given example.   Personally, I try to understand them once they've been enshrined (as here), but also try to avoid new shrine-building (as in the case of "rival").

**Comment by arademaker** [comment]
thank you @restinplace for your comments but the issue is much more specific. IMHO, we should find another place for broad discussions and for the issues, we need to be more direct towards decisions.

That said if we accepted the nominalization of `destruction` as having 3 possible readings (event, act, and result/state), then we must accept that sense event must not contain the parenthetical remark. On the other hand, we can understand that result and state are different enough to justify the merging of event/result and leaving state and act as separated senses.

In a more broad discussion, first I would like to explore how regular is WN in the modeling of such cases - and other related morphological derivations. For instance, `destruction@n` is derivated related to `destroy@v` which is troponym of `subvert@v` which is derivated related to `subversion@n` which in turn, must have some relation with `destruction@n`...  Second, I would like to discuss how regularities could be modeled in a more compact representation avoiding the unnecessary explicit enumeration of senses. The point is, as always, how to distinguish situations in WN that are mistakes -- common to any large project -- from the ones that are deliberate decisions.

**Comment by jmccrae** [comment]
@arademaker so the issue here is the removal of the phrase "(or the result of an event)" from the synset `ewn-07349090-n`?

**Comment by arademaker** [comment]
yes, to improve the definitions and respecting https://github.com/globalwordnet/english-wordnet/issues/186#issuecomment-524625885 comment.
